megalodon/src/gotosocial.ts

```diff
@@ -709,7 +709,7 @@ export default class Gotosocial implements MegalodonInterface {
         comment: note
       })
     }
-    return this.client.post<GotosocialAPI.Entity.Relationship>(`/api/v1/accounts/${id}/note`).then(res => {
+    return this.client.post<GotosocialAPI.Entity.Relationship>(`/api/v1/accounts/${id}/note`, params).then(res => {
```

Comment:
Note: This is a logic change. I assumed it was accidental that `params` was unused, and added it to the POST call

Reply:
Probably, the previous logic was wrong. The API receives comments in parameters.
https://docs.joinmastodon.org/methods/accounts/#note